src/app/core/feedback/feedback.service.ts

```diff
@@ -75,4 +75,14 @@ export class FeedbackService {
 				})
 			);
 	}
+
+	updateFeedbackAssignee(feedbackId: string, assignee: string): Observable<Feedback> {
```

Comment:
@sshah-asymmetrik can we please add some test coverage for these new service methods?